tests/test_bootstrap.py

```diff
@@ -261,3 +261,62 @@ def test_write_constraints_file_duplicates():
         d==1.0
         """).lstrip()
     assert expected == buffer.getvalue()
+
+
+def test_write_constraints_file_multiples():
+    buffer = io.StringIO()
+    raw_graph = {
+        "": {
+            "download_url": "",
+            "pre_built": False,
+            "version": "0",
+            "canonicalized_name": "",
+            "edges": [
+                {
+                    "key": "docling==2.7.0",
```

Comment:
I am not sure whether we should use actual names of packages that we build and instead switch to generic names like a, b, foo, bar, etc.

Apart from that, this looks good!

Reply:
That's a good point, I should be consistent with the other tests.